lib/std/fmt.zig

```diff
@@ -391,13 +401,30 @@ test parseInt {
     try std.testing.expectEqual(@as(i5, -16), try std.fmt.parseInt(i5, "-10", 16));
 }
 
+/// Parses an integer with a specified `sign`, accepting a generic `Character`
+/// type for width and a buffer of `Character`s, in the specified `base` of an
+/// integral type of value `Result`
+///
+/// When `base` is zero the string prefix is examined to detect the true base:
+///  * A prefix of "0b" implies base=2,
+///  * A prefix of "0o" implies base=8,
+///  * A prefix of "0x" implies base=16,
+///  * Otherwise base=10 is assumed.
+///
+/// Ignores '_' character in `buf`.
```

Comment:
I don't think doc comments need to be copied from public functions to private helpers.